addOns/reports/src/main/zapHomeFiles/reports/traditional-html-plus/report.html

```diff
@@ -518,6 +518,116 @@ <h3 th:text="#{report.alerts.detail}">Alert Detail</h3>
 			<div class="spacer"></div>
 		</th:block>
 	</th:block>
+
+	<th:block th:if="${reportData.isIncludeSection('sequencedetails')}">
+		<th:block
+			th:if="${reportData.reportObjects.get('seqAScanData') != null}">
+			<h3 th:text="#{report.sequences.summary.name}">Summary of
+				Sequences</h3>
+			<sup>For each step: result (Pass/Fail) - risk (of highest
+				alert(s) for the step, if any).</sup>
+
+			<th:block
+				th:each="seq, seqState: ${reportData.reportObjects.get('seqAScanData').seqData}">
+				<table class="alerts">
+					<tr>
+						<td th:text="${seq.sequenceName}">sequenceName</td>
+						<th:block th:each="step, stepState: ${seq.getSteps()}">
+							<td th:class="(${step.isPass()} ? 'pass' : 'fail')"
+								th:with="riskLvl=(${step.getHighestAlert()})"><span
+								th:text="(${step.isPass()} ? 'Pass' : 'Fail')">isPass</span> <span
+								th:if="${riskLvl &gt; 0}"
+								th:class="'alert-' + ${step.getHighestAlert() + 'a'}"
+								th:text="' - '">flag</span> <span
+								th:text="(${riskLvl &gt; 0} ? ${helper.getRiskString(riskLvl)} : '')">risk</span>
+							</td>
+						</th:block>
+					</tr>
+				</table>
+			</th:block>
+
+			<div class="spacer-lg"></div>
+
+			<th:block th:if="${reportData.isIncludeSection('sequencedetails')}">
+				<th:block
+					th:if="${reportData.reportObjects.get('seqAScanData') != null}">
+					<h3 th:text="#{report.sequences.details.name}">Sequences
+						Details</h3>
+					<sup>With the associated active scan results.</sup>
+
+					<th:block
+						th:each="seq, seqState: ${reportData.reportObjects.get('seqAScanData').seqData}">
+						<h4 th:text="${seq.sequenceName}">sequenceName</h4>
+						<div class="lm2">
+							<th:block th:each="step, stepState: ${seq.getSteps()}">
+
+								<h4
+									th:text="'Step ' + ${stepState.index + 1} + ': ' + ${step.getOriginalMsg().getRequestHeader().getMethod()} + ' - ' + ${step.getOriginalMsg().getRequestHeader().getURI().toString()}">stepDesc</h4>
+
+								<div class="lm2">
+
+									<span th:text="#{report.sequences.step.result}">Result</span> <span
+										th:text="${step.getResult()}">result</span><br> <span
+										th:if="${step.getAlerts().size() &gt; 0}"
+										th:text="#{report.sequences.step.alerts}">Alerts</span>
+									<div class="lm2" th:if="${step.getAlerts().size() &gt; 0}">
+										<th:block th:each="alert: ${step.getAlerts()}">
+											<span th:class="'alert-' + ${alert.getRisk() + 'b'}">&nbsp;<span>
+													<a th:href="'#plugin-' + ${alert.getPluginId()}"
+													th:text="${alert.getName()}" href="#plugin-pluginId">Alert
+														Name</a> <!-- <span th:text="${alert.getName()}">alertName</span> -->
+											</span></span>
+										</th:block>
+									</div>
+
+									<table>
+										<th colspan="2" th:text="#{report.sequences.step.original}">Original</th>
```

Comment:
This is black on dark grey in light mode, so v difficult to read.

Reply:
Switched the table classes. Should be better now.